packages/@guardian/source-react-components-development-kitchen/components/divider/README.md

```diff
@@ -54,3 +54,9 @@ How much space is given above the divider
 **`string`**
 
 This text is centered inside the dividing line
+
+### `cssOverrides`
+
+**`SerializedStyles | SerializedStyles[] | undefined`**
+
+An escape hatch, used to override default styles if needed.
```

Comment:
To avoid duplicating API docs, we've been following the pattern of linking to Storybook:

```md
## Use

### API

See [storybook](https://guardian.github.io/source/?path=/docs/kitchen-source-react-components-development-kitchen-divider--playground)

### How to use

For context and visual guides relating to usage see the [Source Design System website](https://theguardian.design).
```

Could we remove all props documentation from this README?

Reply:
sounds good - will do